autoload/sj/php.vim

```diff
@@ -187,8 +187,24 @@ function! s:JoinList(start_char, end_char)
   return 1
 endfunction
 
+function! s:SplitNextArrow()
+  let l:arrow_or_paren = search('\v(\(|\S\zs-\>\ze)', '', line('.'))
+
+  if ! arrow_or_paren
+    return
+  endif
+
+  if matchstr(getline('.'), '\%' . col('.') . 'c.') == '('
+    normal! %
+  else
+    normal! i

+  endif
+
+  call s:SplitNextArrow()
+endfunction
+
 function! sj#php#SplitMethodChain()
-  let pattern = '->\%(\k\+\)'
```

Comment:
I had an issue with this pattern as I have `-` registered as a keyword in my vimrc

Reply:
This is fine, and it seems to work well in this case, but I would generally advise you to not add `-` to `iskeyword` :). It breaks *a lot* of things.

Reply:
I realised that...